configure.ac

```diff
@@ -577,6 +577,9 @@ AC_CONFIG_FILES([
     ipaserver/Makefile
     ipatests/Makefile
     ipatests/man/Makefile
+    packaging/Makefile
```

Comment:
can we rename this to `wheel_packaging` or `pypi_packaging` to be clear what is the purpose, because it is only for `bdist_wheel`